pingpong/test_multi_message.py

```
@@ -14,34 +14,42 @@
 pytestmark = pytest.mark.asyncio
 
 
-async def _setup_handler_with_initial_message(db):
+async def _create_handler_context(
```

Ruff (0.15.5)

[warning] Missing return type annotation for private function `_create_handler_context`

(ANN202)